setup.py

```diff
@@ -59,7 +59,7 @@ def get_advanced_templates():
         'Flask',
         'flask-compress',
         'urllib3==1.25.10',
-        'chardet',
+        'chardet<4.0',
```

Comment:
https://github.com/psf/requests/pull/5688 so until we bump requests, we need to use less than 4.